### PR TITLE
fix bug: passing namespace as domain

### PR DESCRIPTION
**Pull request by rokii**

### CODE REVIEW

**Comment by rokii** [comment]
@rshriram  The domainSuffix is just set to model.ConfigMeta.Domain,https://github.com/istio/istio/blob/master/pilot/pkg/config/kube/ingress/conversion.go#L246-L247

**Comment by costinm** [comment]
Comments ? Example ? Hard to follow.

**Comment by rokii** [comment]
@costinm Thx for pointing out this. I created a issue and link to this PR. Resolving this https://github.com/istio/istio/issues/6778
